crates/python_extractor/src/main.rs

```diff
@@ -123,12 +123,16 @@ async fn main() -> Result<()> {
 }
 
 fn expand_path_to_defs_from_offset(from_given_path: &str, path: &str) -> Vec<String> {
-    if let Some(rem) = path.strip_prefix(from_given_path) {
+    if let Some(rem0) = path.strip_prefix(from_given_path) {
+        let rem = match rem0.strip_prefix("site-packages/") {
```

Comment:
This is a minor change on Python extractor, needed because rules_python Bzlmod support uses pip-tools, which I think places the 3rdparty source files inside a `site-packages/` directory, per module.